blazor/spreadsheet/undo-redo.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: post
+title: "Undo Redo in Blazor Spreadsheet Component"
+component: "Spreadsheet"
+description: "Learn how to use the undo and redo functionality in the Syncfusion Blazor Spreadsheet component."
+platform: Blazor
+control: Spreadsheet
+documentation: ug
+---
+
+# Undo Redo in Spreadsheet
+
+<b>Undo</b> option helps you to undone the last action performed and <b>Redo </b>option helps you to do the same action which is reverted in the Spreadsheet. 
+
+
+## Undo
+
+It reverses the last action you performed with Spreadsheet. <b>Undo</b> can be done by any of the following ways:
+
+* Select the undo item from HOME tab in Ribbon toolbar.
+
+* Use <kbd>Ctrl</kbd> + <kbd>Z</kbd> keyboard shortcut to perform the undo.
+
```

Comment:
Attach Screenshot for Undo button alone
![image](https://github.com/user-attachments/assets/f4b0555e-0cb3-47b5-becb-9d03be8d6918)